app/views/spree/components/products/_product_form.html.erb

```diff
@@ -0,0 +1,63 @@
+<%
+  html = {
+    itemprop: 'offers',
+    itemscope: true,
+    itemtype: 'http://schema.org/Offer'
+  }
+%>
+
+<%= form_for :order, url: populate_orders_path, html: html do |f| %>
+  <%= render partial: 'spree/components/products/product_variants' %>
+
+  <% if @product.price_for(current_pricing_options) and !@product.price.nil? %>
+    <div class="product-submit" id="product-price">
+      <h2 class="product-submit__title">
+        <span
+          class="price selling"
+          itemprop="price"
+          content="<%= @product.price_for(current_pricing_options).to_d %>"
+        >
+          <%= display_price(@product) %>
+        </span>
+        <span
+          itemprop="priceCurrency"
+          content="<%= current_pricing_options.currency %>"
+        ></span>
+      </h2>
+
+      <% if @product.master.can_supply? %>
+        <link itemprop="availability" href="http://schema.org/InStock" />
```

Comment:
What about moving this availability content into a separate component?

Reply:
This block of code is used only in this component, maybe in this case it's a bit overkill, as it is also strictly related to the component itself.

Reply:
I'm ok with this but I also think that it's in this component for layout reasons at the moment. You can theoretically move it anywhere on the page, isn't it? Or use it into the products listing as well. 

For users that want to make things like that it will be much easier to just use the component when they want if it's a separate file.

Anyway, I don't want this to be a blocker, we can even extract it later if we think it makes more sense. Up to you!